packages/ui5-application-writer/src/data/defaults.ts

```diff
@@ -57,10 +57,13 @@ export const enum UI5_DEFAULT {
     SAPUI5_CDN = 'https://ui5.sap.com',
     OPENUI5_CDN = 'https://openui5.hana.ondemand.com',
     TYPES_VERSION_SINCE = '1.76.0',
+    TYPES_VERSION_BEST_MIN = '1.102.0',
     TYPES_VERSION_PREVIOUS = '1.71.15',
     TYPES_VERSION_BEST = '1.108.0',
+    V2_FE_TYPES_AVAILABLE = '1.108.0',
     ESM_TYPES_VERSION_SINCE = '1.94.0',
     MANIFEST_VERSION = '1.12.0',
+    UI5_VERSION_VARIABLE = '${sap.ui5.dist.version}',
```

Comment:
This string is S/4 pipeline specific. It should never be "seen" by the open ux writers. Instead the value should be replaced by the caller of the writer.

Reply:
👍 removed in https://github.com/SAP/open-ux-tools/pull/845/commits/53dc7bccc45149585893580868b322ed87039676